sklearn/decomposition/dict_learning.py

```diff
@@ -146,8 +146,11 @@ def _sparse_encode(X, dictionary, gram, cov=None, algorithm='lasso_lars',
 
     elif algorithm == 'omp':
         # TODO: Should verbose argument be passed to this?
-        new_code = orthogonal_mp_gram(gram, cov, regularization, None,
-                                      row_norms(X, squared=True),
+        new_code = orthogonal_mp_gram(Gram=gram, Xy=cov,
```

Comment:
nitpick: Maybe a newline after ``(`` (and different indentation) would make this prettier?